plugins/usdc_coinbase/usdc_coinbase.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"autonity-oracle/plugins/common"
+	"autonity-oracle/types"
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/go-hclog"
+	"io"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	version = "v0.0.1"
+	path    = "v2/prices/USDC-USD/spot"
+)
+
+var defaultConfig = types.PluginConfig{
+	Name:               "usdc_coinbase",
+	Key:                "",
+	Scheme:             "https",
+	Endpoint:           "api.coinbase.com",
+	Timeout:            10,  // 10s
+	DataUpdateInterval: 120, // 120s
```

Comment:
This is the best update interval for the free API, if I  remember right? Just resolve if I'm right!

Reply:
> Question: DataUpdateInterval default values. Set to a default of 30 for the FX and CAX plugins. Set to 120 for the crypto plugins for USDC.
> 
> Is 120 the best update interval for the USDC APIs? Else, why not harmonise all plugins to the same default update frequency for consistent sampling if the operator leaves all at default setting?

Resolved in db8d585f44b1239ec982ffdf09074a16447466a0 where I created a single node e2e test to verify if 30s interval works. In the last e2e testing, I created 4 oracle servers within a single host (single IP address), thus I extend the interval from 30s to 120s to prevent from the rate limits. With the new test, 30s interval in a single client works good. Now we restore 30s as the default setting.